ada_feeding_perception/ada_feeding_perception/face_detection.py

```diff
@@ -327,11 +325,13 @@ def depth_callback(self, msg: CompressedImage):
                     desired_encoding="passthrough",
                 )
 
-                # Retrieve the depth value averaged over all mouth coordinates
+                # Retrieve the depth value averaged over a 9x9 pixel square around
+                # point between eyes
                 depth_sum = 0
-                for point in img_mouth_points:
-                    depth_sum += closest_depth[int(point[1])][int(point[0])]
-                depth = depth_sum / float(len(img_mouth_points))
+                for x in range(u - 4, u + 4):
+                    for y in range(v - 4, v + 4):
+                        depth_sum += closest_depth[int(x)][int(y)]
+                depth = depth_sum / float(81)
```

Comment:
I don't like this magic number. I know it comes from `9**2`, but don't think it is intuitive to readers. 

Given that some of the (x,y) pairs will get rejected for being out-of-bounds anyway, you also can't assume that all 81 points will be in the `depth_sum`. Therefore, I'd just recommend enumerating the points in the depth_sum, and using that number to do the average.